Simplified/NYPLBookDetailViewController.m

```diff
@@ -19,6 +19,7 @@ @interface NYPLBookDetailViewController () <NYPLBookDetailViewDelegate, NYPLProb
 
 @property (nonatomic) NYPLBook *book;
 @property (nonatomic) NYPLBookDetailView *bookDetailView;
+@property (nonatomic, retain) NYPLNetworkExecutor *executor;
```

Comment:
minor: `retain/`strong` is unneeded since it's the default (also I think `retain` is deprecated unless you're not using ARC?)

Reply:
Fixed